templates/zerver/for/open-source.md

```diff
@@ -0,0 +1,99 @@
+A great open source project needs a great group chat service.  Here's
+why we think Zulip is the best choice you can make today.
+
+(Psst, I'll spoil the ending: not only is Zulip itself open source,
+but hosting on zulipchat.com is free for open source projects.
+[Sign up now](https://zulipchat.com/beta/)!)
+
+## Great for developers
+
+* Messages take Markdown formatting, [including code
+  blocks](/help/format-your-message-using-markdown#code).  Super easy
+  to discuss code, paste error messages, etc.
+
+* Want to refer to issues or code reviews with short notation like
+  `#1234`, `T1234` -- or any other style of your own?  Set up any
```

Comment:
Why do we use spaces around double hyphens rather than a full---made with three hyphens in a row in Markdown---em dash in most/all of our documentation?